docs/nidcpower/class.rst

```diff
@@ -858,6 +871,50 @@ get_channel_name
 
 
 
+get_channel_names
```

Comment:
This shouldn't be in the diff for the PR.

Reply:
135b23825d39fa1ed4dde940deb59ffef523bb27